core/src/replay_stage.rs

```diff
@@ -1755,9 +1755,10 @@ impl ReplayStage {
 
     pub fn get_unlock_heaviest_subtree_fork_choice(operating_mode: OperatingMode) -> Slot {
         match operating_mode {
-            OperatingMode::Development => std::u64::MAX / 2,
+            OperatingMode::Development => 5000,
```

Comment:
Why not 0?

Reply:
ah I was just following the trend from the Nonce unlock, is devnet restarted from genesis?

Reply:
It is persistent so 5000 is effectively as good as 0.  But 0 is more clear IMO.   We probably can set Nonce unlock back to 0 now too

Reply:
agreed, changed!